### ISSUE TITLE
Use amount values based on Neo X decimals in deposit events

### ISSUE THREAD

**Issue by Tobeyw** [bane-labs/bridge-evm-contracts#206]
https://github.com/bane-labs/bridge-evm-contracts/blob/04a82749cab7aebeef08ee333c2c6176a1be436a/contracts/bridge/BridgeImpl.sol#L129-L160

Can the `amount` parameter in the `gasDeposit/GasClaimable ` event be replaced with `sendvalue`? It is easy to confuse when we display data in the browser.

**Comment by mialbu** [comment]
Yes, we can. Just confirming: You'd like that _depositEntry.amount_ is replaced with _sendValue_ in the event, so that the value with 18 decimals and not the one with 8 decimals will be emitted, correct?
@Tobeyw

**Comment by Tobeyw** [comment]
> Yes, we can. Just confirming: You'd like that _depositEntry.amount_ is replaced with _sendValue_ in the event, so that the value with 18 decimals and not the one with 8 decimals will be emitted, correct? @Tobeyw

Yes, it is best to unify the accuracy according to `neox`,not `neoN3`

**Comment by mialbu** [comment]
I agree, we should change this appropriately. I'll create a new milestone and add it there to be included in the next upgrade.
